spm-sender/src/main/java/com/sematext/spm/client/sender/flume/influx/HttpInfluxClient.java

```diff
@@ -140,7 +140,8 @@ public boolean isTrusted(X509Certificate[] arg0, String arg1) throws Certificate
             .setDefaultCredentialsProvider(credsProvider)
             .build();
 
-        HttpHost proxy = new HttpHost(proxyContext.getHost(), proxyContext.getPort(), useHttps ? "https" : "http");
+        HttpHost proxy = new HttpHost(proxyContext.getHost(), proxyContext.getPort(), 
```

Comment:
It probably makes sense to purge the `useHttps` variable from this file, since it was previously used for the same purpose that you now introduced a config param for, only the decision was made based on `baseServerUrl.contains("https://")`.